luigi/interface.py

```diff
@@ -181,6 +184,21 @@ class PidLockAlreadyTakenExit(SystemExit):
     pass
 
 
+def _detailed_summary(func):
```

Comment:
Wow, I'm impressed by your Python fu.

But! I did ask for something **simpler**, something easier to read. I meant you to inline this implementation into the `build`  and `run` function (if you decide to do this for `run` although I've tried to deprecate it.).

I wouldn't be surprised if you can inline this using a simple `if` statement or maybe even `if` expression.

Reply:
Alright! Used an `if` expression now 😄